bazel/envoy_build_system.bzl

```diff
@@ -0,0 +1,41 @@
+# References to Envoy external dependencies should be wrapped with this function.
+def envoy_external_dep_path(dep):
+    return "//external:%s" % dep
+
+ENVOY_COPTS = [
+    "-ggdb3", "-fno-omit-frame-pointer", "-Wall", "-Wextra", "-Werror",
```

Comment:
`-ggdb3` shouldn't be enforced here, you can include it in .bazelrc (but only for specific config). bazel has `-c` flag to control fastbuild vs debug (dbg) vs release (opt)